app/modules/sightings/resources.py

```diff
@@ -425,7 +454,29 @@ def patch(self, args, sighting):
             )
 
         if edm_count > 0:
+            cleanup = SightingCleanup()
             log.debug(f'wanting to do edm patch on args={args}')
+
+            # we pre-check any annotations we will want to attach to new encounters
+            enc_anns = (
+                []
+            )  # list of lists of annotations for each encounter (if applicable)
+            try:
+                for arg in args:
+                    if (
+                        arg.get('path', None) == '/encounters'
```

Comment:
so if this patch is adding an annotation to an encounter, why is it implemented on the sighting rather than the encounter?

Reply:
this patch adds an _encounter_ to the sighting.   this code allows this (new, added) encounter to have (optional) annotations referenced.